src/escpos/escpos.py

```diff
@@ -711,13 +712,14 @@ def control(self, ctl, pos=4):
         elif ctl.upper() == "CR":
             self._raw(CTL_CR)
         elif ctl.upper() == "HT":
-            if not (1 <= pos <= 16):
+            if not (0 <= count <= 32 and 1 <= tab_size <= 255 and count * tab_size < 256):
```

Comment:
Sounds also good to me, I am just not 100% sure how to do the indentations :-) But a good editor should help you.